can_tools/scrapers/base.py

```diff
@@ -51,37 +51,32 @@ def extract_CMU(
         """
         foo = df[var_name]
         return df.assign(
-            **{
-                c: foo.map(lambda x: cmu[x].__getattribute__(c))
-                for c in columns
-            }
+            **{c: foo.map(lambda x: cmu[x].__getattribute__(c)) for c in columns}
         )
 
     @abstractmethod
-    def put(self, conn, df):
+    def put(self, conn: sa.engine.base.Engine, df: pd.DataFrame):
```

Comment:
Am I wrong that most of our `put` methods take a string rather than an engine?

Reply:
good catch, fixed